Seminar-IS/Seminar02/Pract 02/task02.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+
+void readSourceCode() {
+    ifstream inFile("result.txt");
+    if (!inFile.is_open()) {
+        return;
+    }
+
+    size_t small_letters = 0;
+    size_t big_letters = 0;
+    size_t digits = 0;
+    size_t other = 0;
+    char symbol = inFile.get();
```

Comment:
You can use do-while and avoid reading the first character out of the while loop.

Reply:
Using do-while, I would have to add a separate check in order to ignore the `EOF` character. Otherwise it counts it as `other`.